conf/test.config

```diff
@@ -35,6 +35,7 @@ process {
     publishDir = { "${params.outdir}/pipeline_info" }
```

Comment:
Not sure why this information is here. Move to `conf/modules.config`.

Reply:
This is already in `conf/modules.config`

```
publishDir = [
        path: { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
        mode: params.publish_dir_mode,
        saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
    ]
```

Does it need to be changed?

Reply:
You don't need the `publishDir` and `withName: SAMTOOLS_VIEW` sections in both `test.config` and `modules.config`. I would say better to keep it in `modules.config` and delete it in `test.config`. Also take a look at the output directory and make sure it only includes files we want to save, not all results from `work`.